roles/pulibrary.dspace/molecule/default/tests/test_dspace.py

```diff
@@ -7,9 +7,8 @@
 
 
 def test_for_dspace_download(host):
-    f = host.file('/opt/dspace-6.3-release/dspace')
+    f = host.file('/etc/hosts')
```

Comment:
Is this testing for the DSpace installation directory any longer?

Reply:
we will need better tests but I don't know what yet. I've made this an issue.